### ISSUE TITLE
Add a reset feature

### ISSUE THREAD

**Issue by rgaudin** [offspot/content-filter#1]
Settings page would benefit from a reset/clear option

**Comment by QueTeddy** [comment]
Could I give this a trial?

**Comment by rgaudin** [comment]
Sure. Send a PR or ask questions here if you're not sure about how to do it.

**Comment by Laxmilashkare** [comment]
> Sure. Send a PR or ask questions here if you're not sure about how to do it.

Hello sir, i would like to work on this issue can you guide me to solve this issue

**Comment by rgaudin** [comment]
@Laxmilashkare get familiar with the tools and you should understand the need and how to implement it. Let me know if you have more specific questions

**Comment by Laxmilashkare** [comment]
@rgaudin okay thank you

**Comment by Laxmilashkare** [comment]
Okay Thank you!

**Comment by Laxmilashkare** [comment]
@rgaudin Hello Sir, I need the admin credentials (username & password) for Caddy's Admin API (port 2020) to proceed. I am stuck due to authentication. Could you please share them?

**Comment by rgaudin** [comment]
There is none, see [caddy doc](https://caddyserver.com/docs/caddyfile/options#admin)

**Comment by 0xCrimsonSky** [comment]
@rgaudin is this feature still required? If yes, can I take this up?

**Comment by rgaudin** [comment]
@0xCrimsonSky yes it's still required but there's no user of this at the moment.

**Comment by 0xCrimsonSky** [comment]
> [@0xCrimsonSky](https://github.com/0xCrimsonSky) yes it's still required but there's no user of this at the moment.

I can probably work on this as it's still an open one. Is it okay with you @rgaudin ? I have an idea of what to do.

**Comment by rgaudin** [comment]
@0xCrimsonSky sure. I am assigning it to you.
